.zombienet-tests/0001-custom.js

```diff
@@ -0,0 +1,17 @@
+const polkadotApi = require("@polkadot/api");
+
+async function connect(apiUrl, types) {
+    const provider = new polkadotApi.WsProvider(apiUrl);
+    const api = new polkadotApi.ApiPromise({ provider, types });
+    await api.isReady;
+    return api;
+}
+
+async function run(nodeName, networkInfo, args) {
+    const {wsUri, userDefinedTypes} = networkInfo.nodesByName[nodeName];
+    const api = await connect(wsUri, userDefinedTypes);
```

Comment:
replace:
```javascript
    const api = await connect(wsUri, userDefinedTypes);
```
with:
```javascript
    const api = await connect(JSON.strigify(customChainSpec), userDefinedTypes);  
```

Reply:
`customChainSpec   should be an `string`?